storage-resize-images/extension.yaml

```diff
@@ -155,3 +155,12 @@ params:
       If you prefer not to overwrite or add `Cache-Control` metadata, leave this field empty.
     example: max-age=86400
     required: false
+
+  - param: IMAGE_TYPE
+    label: Image type to convert to
```

Comment:
Can you make this a "select" param instead? Then the user can select from a list instead of typing the file type. Also please include one option which is "do not convert"